masonry/src/doc/creating_app.md

```diff
@@ -13,10 +13,6 @@
 </div>
 
 
-<!-- TODO - This doc is located in masonry, but gives a tutorial for Masonry. -->
-<!-- Because the `masonry` doc has several links to this folder, -->
-<!-- we can't easily move it to `masonry_winit`. We need to sort that out. -->
-
```

Comment:
There's been a few crate splits since this was written. Docs have now been split between crates, so we did sort it out.